extensions/wikia/WAMPage/css/WAMPage.scss

```diff
@@ -147,12 +148,16 @@ $color-wam-button: #092344;
 			position: absolute;
 			text-decoration: none;
 			width: 144px;
+
 			figure {
-				background: url('/extensions/wikia/WAMPage/images/wam-imgplaceholder.png'); /* inline */;
+				background-image: url('/extensions/wikia/WAMPage/images/placeholder.svg'); /* inline */
+				background-position: center;
+				background-repeat: no-repeat;
```

Comment:
Can't do it cause it will overwrite a background-color property from another selector. :)